frontend/src/components/module-viewer/content/blocks/FlipCard.tsx

```diff
@@ -0,0 +1,35 @@
+import { Center, SimpleGrid } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+const FlipBlock = ({
+  front,
+  back,
+}: {
+  front: string;
+  back: string;
+}): React.ReactElement => {
+  const [flipped, setFlipped] = useState(false);
+
+  return (
+    <Center className="card-container">
+      <SimpleGrid
+        w="100%"
+        maxW="400px"
+        minH="10rem"
+        padding="2rem"
+        borderTopColor="brand.royal"
+        borderTopWidth="7px"
+        borderRadius="4px"
+        boxShadow="lg"
+        columns={1}
+        className={`card ${flipped ? "card-flipped" : ""}`}
+        onClick={() => setFlipped(!flipped)}

```

Comment:
Again might be nice to use the functional form in case users click very quickly on a slow device.